src/Core/src/Platform/iOS/ImageViewExtensions.cs

```diff
@@ -54,5 +55,39 @@ public static void UpdateSource(this UIImageView imageView, UIImage? uIImage, II
 				imageView.Image = uiImage;
 			}, scale, cancellationToken);
 		}
+
+		/// <summary>
+		/// Gets the size that fits on the screen for a <see cref="UIImageView"/> to be consistent cross-platform.
+		/// </summary>
+		/// <remarks>The default iOS implementation of SizeThatFits only returns the image's dimensions and ignores the constraints.</remarks>
+		/// <param name="imageView">The <see cref="UIImageView"/> to be measured.</param>
+		/// <param name="constraints">The specified size constraints.</param>
+		/// <returns>The size where the image would fit depending on the aspect ratio.</returns>
+#pragma warning disable RS0016 // Add public types and members to the declared API
+		public static CGSize SizeThatFitsImage(this UIImageView imageView, CGSize constraints) // Method name subject to change
```

Comment:
Looking for feedback on this method name! Once that's set I'll remove the pragma and add it to the api txt.